addons/web/controllers/main.py

```diff
@@ -815,13 +815,24 @@ def _write_group_header(self, row, column, label, group, group_depth=0):
 
         label = '%s%s (%s)' % ('    ' * group_depth, label, group.count)
         self.write(row, column, label, self.header_bold_style)
+        decimal_places = 2
+        if any(f.get('type') == 'monetary' for f in self.fields[1:]):
+            decimal_places = max(res['decimal_places'] for res in group._model.env['res.currency'].search_read([], ['decimal_places']))
```

Comment:
Just thought about this: in case there would be no active currency (not sure if it is even possible), it could crash. I think is doesn't cost much to do:

```suggestion
            decimal_places = res['decimal_places'] for res in group._model.env['res.currency'].search_read([], ['decimal_places'])
            decimal_places = max(decimal_places) if decimal_places else 2
```